src/Components/App/App.test.js

```diff
@@ -1,9 +1,41 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { shallow } from 'enzyme'
 import App from './App';
+import newObjs from './App.helper'
+import peopleData from '../../Data/PeopleTestData'
 
-it('renders without crashing', () => {
-  const div = document.createElement('div');
-  ReactDOM.render(<App />, div);
-  ReactDOM.unmountComponentAtNode(div);
-});
+describe('App', () => {
+  let wrapper;
+  beforeEach(() => {
+    wrapper = shallow(<App />)
+  })
+
+  it('renders without crashing', () => {
+    const div = document.createElement('div');
+    ReactDOM.render(<App />, div);
+    ReactDOM.unmountComponentAtNode(div);
+  });
+
+  it('should have be able to store people data in state', () => {
+    expect(wrapper.state().people).toEqual([])
+  })
+
+  it('should be able to store planet data', () => {
+    expect(wrapper.state().planets).toEqual([])
+  })
+
+  it('should be able to store vehicle data', () => {
+    expect(wrapper.state().planets).toEqual([])
+  })
+})
```

Comment:
Could we combine these three tests since they seem to be testing similar things with state?